conjure-go-client/httpclient/config.go

```diff
@@ -106,6 +106,7 @@ type SecurityConfig struct {
 	CAFiles  []string `json:"ca-files,omitempty" yaml:"ca-files,omitempty"`
 	CertFile string   `json:"cert-file,omitempty" yaml:"cert-file,omitempty"`
 	KeyFile  string   `json:"key-file,omitempty" yaml:"key-file,omitempty"`
+	CAs      []string `json:"cas"`
```

Comment:
Lets add comment(s) explaining that these are PEM-encoded content and also fields for cert and key which we will need as well for client auth

Reply:
Sorry I didn't mean to include this, removed